pkg/services/featuremgmt/toggles_gen.json

```diff
@@ -1231,6 +1231,31 @@
         "hideFromAdminPage": true
       }
     },
+    {
+      "metadata": {
+        "name": "enableSCIM",
+        "resourceVersion": "1730980484343",
+        "creationTimestamp": "2024-11-07T11:54:44Z"
+      },
+      "spec": {
+        "description": "Enables SCIM support for user and group management",
+        "stage": "experimental",
+        "codeowner": "@grafana/identity-access-team"
+      }
+    },
+    {
+      "metadata": {
+        "name": "enableScim",
+        "resourceVersion": "1730976932709",
+        "creationTimestamp": "2024-11-07T10:55:32Z",
+        "deletionTimestamp": "2024-11-07T11:54:44Z"
+      },
+      "spec": {
+        "description": "Enables SCIM support for user and group management",
+        "stage": "experimental",
+        "codeowner": "@grafana/identity-access-team"
+      }
+    },
```

Comment:
Left over from the initial name?
```suggestion
```

Reply:
Ohh wow I missed this, but you are right! Thanks for the catch

Reply:
Removed.